test/Interop/Cxx/members/emit-code-for-nested-struct-irgen.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+
+import MethodCallsMethodFromNestedStruct
+
+public func getValueFromMethod() -> CInt {
+  return callMethod(41)
+}
+
+// CHECK: define linkonce_odr i32 @_ZN13IncrementUser11Incrementor9incrementEi(%"struct.IncrementUser::Incrementor"* %this, i32 %t)
```

Comment:
Just FYI this (and similar checks) will be different on Windows (that's probably why the CI is failing).

Reply:
Done, thanks!